docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -67,6 +67,40 @@ A filter that applies to the result of a block's `save` function. This filter is
 
 The filter's callback receives an element, a block type and the block attributes as arguments. It should return an element.
 
+This example below wraps the existing save element inside of a wrapper element.
+
+_Example:_
+
+Adding a wrapper to core quote blocks
+
+{% codetabs %}
+{% ES5 %}
+```js
+var createElement = wp.element.createElement;
+var quoteSaveFilter = ( el, type, attributes ) => {
+  if ( type.name === "core/quote" ) {
+    var newEl = createElement( "div", { className: "MYWRAPPER" }, el );    
+    return newEl;
+  }
+  return el;
+};
+
+wp.hooks.addFilter( "blocks.getSaveElement", "my-plugin/quote-save", quoteSaveFilter );
+```
+{% ESNext %}
+```js
+const quoteSaveFilter = ( el, type, attributes ) => {
+  if ( type.name === "core/quote" ) {
```

Comment:
Still a few style issues in this function:

- Single quotes
- Spaces in JSX template

Can also move the code into a JS file in the project to run existing lint tooling against it.